go/chat/utils/utils.go

```diff
@@ -112,17 +112,17 @@ func AggRateLimits(rlimits []chat1.RateLimit) (res []chat1.RateLimit) {
 // ReorderParticipants based on the order in activeList.
 // Only allows usernames from tlfname in the output.
 // This never fails, worse comes to worst it just returns the split of tlfname.
-func ReorderParticipants(ctx context.Context, g libkb.UIDMapperContext, umapper libkb.UIDMapper,
+func ReorderParticipants(m libkb.MetaContext, g libkb.UIDMapperContext, umapper libkb.UIDMapper,
```

Comment:
I think `mctx` rather than `m` is preferred by the more-letters club

Reply:
hehe sounds good